lang.lua

```diff
@@ -59,8 +59,10 @@ end
 -- Sort preferred languages in order of preference
 local cleaned = ngx.re.sub(lang_header, "^.*:", ""):lower() -- Remove header name "Accept-Language:"
 local options = {} -- Sorted table: { en-US: 1, en: 0.8, de-DE: 0.6, de: 0.4 }
+
 -- Create 2 match groups: language and priority
-local iterator, err = ngx.re.gmatch(cleaned, "\\s*([a-z]+(?:-[a-z])*)\\s*(?:;q=([0-9]+(.[0-9]*)?))?\\s*(,|$)", "i")
+local regex = "\\s*([a-z-]*)\\s*(?:;q=([0-9]+(.[0-9]*)?))?\\s*(,|$)"
```

Comment:
Turns out lua doesn't use regex at all but it's own patterns that just look very similar. For reference check http://www.lua.org/manual/5.2/manual.html#6.4.1